Formula/sipp.rb

```diff
@@ -22,4 +21,8 @@ def install
     system "make", "DESTDIR=#{prefix}"
     bin.install "sipp"
   end
+
+  test do
+    assert_equal "SIPp", shell_output("#{bin}/sipp -v | grep SIPp | awk '{ print $1 }'").strip
```

Comment:
Unfortunately, the test fails with that. The "sipp -v" output is a multi-line block of text with the first line being blank and shell_output doesn't return it properly. 

So I had to improvise with:

  test do
    sipp_output = `#{bin}/sipp -v`.split(/$/)
    sipp_output.each { |o| print o }
    assert_match "SIPp v3.5.1", sipp_output[0]
  end

I've commited these changes for review.